src/helpers.cpp

```diff
@@ -77,8 +77,11 @@ namespace ada::helpers {
     }
 
     // Remove path’s last item, if any.
-    if (!path.empty()) {
-      path.erase(path.rfind('/'));
+    for (long i = path.size() - 1; i >= 0; --i) {
```

Comment:
It is not intuitive for me why this change might be faster. The rfind function might be well optimized.